Sources/Tabman/AutoInsetter/AutoInsetter.swift

```diff
@@ -66,24 +66,25 @@ internal final class AutoInsetter {
             if requiredInsetSpec.additionalRequiredInsets != viewController.additionalSafeAreaInsets {
                 viewController.additionalSafeAreaInsets = requiredInsetSpec.additionalRequiredInsets
             }
-        }
-        
-        guard viewController.shouldEvaluateEmbeddedScrollViews() else {
-            return
-        }
-        viewController.forEachEmbeddedScrollView { (scrollView) in
-            let calculator = self.makeInsetCalculator(for: scrollView, viewController: viewController)
-            let executor = InsetExecutor(view: scrollView, calculator: calculator, spec: requiredInsetSpec)
-            
-            executor.execute(store: insetStore)
-            
-            viewController.view.setNeedsLayout()
+        } else { // manual insetting on older versions.
+            guard viewController.shouldEvaluateEmbeddedScrollViews() else {
+                return
+            }
+            viewController.forEachEmbeddedScrollView { (scrollView) in
+                let calculator = self.makeInsetCalculator(for: scrollView, viewController: viewController)
+                let executor = InsetExecutor(view: scrollView, calculator: calculator, spec: requiredInsetSpec)
+    
+                executor.execute(store: insetStore)
+    
+                viewController.view.setNeedsLayout()
+            }
```

Comment:
If I have some calculation logic with insets, then from 2.7.0 release on iOS 11.* and above I should rely on `safeAreaInsets` instead of `contentInset` property?